filebeat/filebeat.yml

```diff
@@ -52,6 +52,9 @@ filebeat:
         # be used.
         #add_error_key: false
 
+      # If chunk_size is set, the input is read in chunks of given byte-length instead of lines
+      # chunk_size: 200
```

Comment:
These changes should be added to `beat.yml` and then run `make update` to apply it to filebeat. The space in front of chunk_size should be removed.